### PR TITLE
diag version

### PR DESCRIPTION
**Pull request by ShreyasRmsft**
**Do not Merge!!!!**

### CODE REVIEW

**Comment by ShreyasRmsft** [comment]
The debug log numbers matched the line numbers at one point of time. Added a bit of code and they got dispalced :p

**Comment by ShreyasRmsft** [comment]
Here's the script im using to upload the logs later

$temp = $Env:Agent_TempDirectory

Write-Host $temp

$filePath = Join-Path $temp "VsTestToolInstallerDiagLog.txt"

Write-Host "##vso[task.uploadfile]$filePath"

File: Tasks/VsTestPlatformToolInstaller/vstestplatformtoolinstaller.ts [hunk 1]
```diff
@@ -6,230 +6,428 @@ import * as os from 'os';
...
+    debugLog('369');
     nugetTool.line('install ' + packageName + ' -Version ' + testPlatformVersion + ' -Source ' + packageSource + ' -OutputDirectory "' + downloadPath + '" -NoCache -DirectDownload');
+    debugLog('371');
 
-    tl.debug(`Downloading Test Platform version ${testPlatformVersion} from ${packageSource} to ${downloadPath}.`);
+    debugLog(`Downloading Test Platform version ${testPlatformVersion} from ${packageSource} to ${downloadPath}.`);
+    debugLog('374');
     let startTime = perf();
+    debugLog('376');
     const resultCode = await nugetTool.exec();
```
Comment (bryanmacfarlane) [inline Tasks/VsTestPlatformToolInstaller/vstestplatformtoolinstaller.ts]: let resultCode:number ??
Should you fail on stdErr:true?  (see shellscript task).  Ask nuget if stderr means error (some tools write status to stderr like git)

Comment (ShreyasRmsft) [reply Tasks/VsTestPlatformToolInstaller/vstestplatformtoolinstaller.ts]: Will add the stderr option once i get this clarified about nuget.exe

**Comment by stephenmichaelf** [comment]
@ShreyasRmsft Can we merge or close this?

**Comment by ShreyasRmsft** [comment]
@stephenmichaelf closing the PR.

**Pull request closed without merging**